bin/build-status

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env perl
+use strict;
+use warnings;
+use 5.010;
+
+use FindBin '$Bin';
+use lib "$Bin/../lib";
+use CPAN2OBS qw/ debug info prompt /;
+use XML::Simple qw/ XMLin /;
```

Comment:
Definitely not the best choice... but i'll assume `XML::Simple` was used already before the refactoring.

Reply:
Yes. The original script is `updateallcpan`.